src/components/simulation/SimulationEventsPanel.svelte

```diff
@@ -98,6 +98,23 @@
       resizable: true,
       sortable: true,
     },
+    span: {
+      field: 'span',
+      hide: true,
+      filter: 'text',
+      headerName: 'Span',
+      resizable: true,
+      sortable: true,
+      valueGetter: params => {
+        if (params.data?.span_id) {
+          const span = $spans.find(span => span.span_id === params.data?.span_id);
```

Comment:
I think you have a $spansMap store that you can use to do a quicker lookup of the span